### PR TITLE
Improve Error Handling when bundles.yml fails

### PR DESCRIPTION
**Pull request by tahmidefaz**
Throws a fatal error when bundles.yml isn't available or when there is a parsing error in the file

### CODE REVIEW

File: config/main.go [hunk 1]
```diff
@@ -97,7 +97,7 @@ func initialize() {
 	options.SetDefault(Keys.Cert, "../test_data/test.cert") // default values of Cert and Key are for testing purposes only
 	options.SetDefault(Keys.Key, "../test_data/test.key")
 	options.SetDefault(Keys.OpenAPISpecPath, "./apispec/api.spec.json")
```
Comment (kruai) [inline config/main.go]: Should this path also be updated?

Comment (tahmidefaz) [reply config/main.go]: @kruai I had difficulty running tests if this path wasn't updated. The path gets switched back to `./bundles/bundles.yml` in `scripts/watch.sh`

File: scripts/watch.sh [hunk 2]
```diff
@@ -9,6 +9,7 @@ else
 fi
 
 export ENT_CA_PATH=./resources/ca.crt
+export ENT_BUNDLE_INFO_YAML=./bundles/bundles.yml
```
Comment (tahmidefaz) [inline scripts/watch.sh]: @kruai Here!

File: config/main.go (hunk 1 above)
Comment (tahmidefaz) [reply config/main.go]: I read that wrong, sorry (blame Monday)! I updated `bundles.yml` path for the above reason. However, tests work just fine without updating API specs path.

Comment (shdunning) [reply config/main.go]: didn't we put a valid bundles in `test_dir`? can't we just use that and not override env vars in 2 places?

File: controllers/subscriptions.go [hunk 3]
```diff
@@ -27,6 +28,9 @@ var bundleInfo []types.Bundle
 func init() {
 	// Reads the yaml file at process startup
 	bundleInfo = GetBundleInfo(config.GetConfig().Options.GetString(config.Keys.BundleInfoYaml))
+	if bundleInfo[0].Error != nil {
+		log.Fatal("Error reading bundles.yml ", zap.Error(bundleInfo[0].Error))
```
Comment (shdunning) [inline controllers/subscriptions.go]: does `log.Fatal` exit the program?

Comment (tahmidefaz) [reply controllers/subscriptions.go]: @shdunning Yeah, it outputs something like this:
```
>> starting...
2019/10/14 14:45:22 Error reading bundles.yml {error 25 0  yaml: line 10: could not find expected ':'}
exit status 1
exited: exit status 1
>> restart backoff... 1000ms
```

File: config/main.go (hunk 1 above)
Comment (tahmidefaz) [reply config/main.go]: @shdunning Yes, we did. However, the init in `subscriptions.go` always gets executed before the tests

**Pull request merged**